ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -24,8 +24,8 @@ def setUp(self):
 
     # will continue with us    
     @parameterized.expand(['save_btn', 'cancel'])
-    @skip('https://modeso.atlassian.net/browse//LIMS-4768')
-    def test001_cancel_button_edit_no(self, save):
+    #@skip('https://modeso.atlassian.net/browse//LIMS-4768')
```

Comment:
why you update This test case in this PR ?

Reply:
Done